Tests/MusicXMLTests/ScoreTests/Reve/manifest.plist

```diff
@@ -0,0 +1,7 @@
+{
+	Description = "First four bars of \"Apres un reve\" excerpted from \"The MIDI-Compatible Part\" tutorial of MusicXML.";
```

Comment:
This is not really that much of a biggie, but I wonder if something like `YAML` would be a slightly nicer format to work with.

It would introduce a dependency (`Yams`), but of all dependencies, this seems like a very robust one.

Reply:
I also have the same feeling that plist is a bit clunky to be a good manifest format. I'll introduce `Yams`